extensions-modules/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "description": "Shared modules for Carbon extensions",
   "dependencies": {
-    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#0.1.2",
+    "dataprotocol-client": "github:Microsoft/sqlops-dataprotocolclient#feat/initFeaturesrelease",
```

Comment:
shouldn't you merge the changes to dataprotocol master prior to merging this...so that other people working in dataprotocol can easily merge the required parallel changes, for example?

Reply:
For sure, I'll merge the other one in and update this before I merge this in, similarly to how we do sqltoolsserivce, but getting the code changes up for review